CHANGELOG.md

```diff
@@ -4,6 +4,121 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.32.1] - 2023-10-17
+
+## 🚀 Features
+
+### Add path to SupergraphRequest ([Issue #4016](https://github.com/apollographql/router/issues/4016))
```

Comment:
This title needs to be changed to refer to `hasNext` not `path`.
e.g.:
Add hasNext to SupergraphResponse.
Note: Response, not Request.